src/test/java/com/joyent/manta/archiver/ManataClientSupplierTest.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) 2017, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.archiver;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Iterator;
+import java.util.Random;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.joyent.http.signature.KeyLoadException;
+import com.joyent.manta.client.MantaClient;
+import com.joyent.manta.client.MantaObject;
+import com.joyent.manta.exception.ConfigurationException;
+import com.joyent.manta.exception.MantaClientHttpResponseException;
+
+import jline.internal.Log;
```

Comment:
This is an immediate red flag for me, why are we using a class in a package with `internal` in the name? My editor tells me the `private static Logger` declared in the class is never used, we should go over how your editor is configured and make sure it shows warnings for unused variables.

Reply:
I had removed the logging lines, I was using it earlier and forgot to remove this.

Reply:
Actually I take that back, Line 168, Log.info(it.next());  But I will remove it. Plus I think the import is probably incorrect, that I think I just let my IDE import.